sdk/tests/conformance/extensions/ext-frag-depth.html

```diff
@@ -140,6 +140,9 @@
             runUniqueObjectTest();
         }
     }
+
+    // Run deferred link tests.
+    runDeferredLinkTests();
```

Comment:
This should have been added in the block above where EXT_frag_depth was successfully enabled. This test can't run properly if the extension isn't supported. See below.